test/e2e/smoke/cluster-workflow-template-whalesay-template-update-parameters.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ClusterWorkflowTemplate
+metadata:
+  name: cluster-workflow-template-whalesay-template
+  labels:
+    workflows.argoproj.io/test: "true"
+spec:
+  entrypoint: whalesay-template
+  arguments:
+    parameters:
+      - name: message
+        value: thanks # message -> thanks
```

```suggestion
        value: thanks # hello world -> thanks
```

the previous message was "hello world"

---

Thanks! ok